ingeniamc/ingenialink-python
# INGK-632 Add an image attribute to the dictionary class

### Description

Store the dictionary's image to an attribute when loading a dictionary.

Fixes #INGK-632

### Type of change

- Add an image attribute to the dictionary class


### Tests
- Connect to a drive with a dictionary.
- Check that the image is stored in the dictionary class.
